packages/driver/src/cypress/error_messages.ts

```diff
@@ -1807,6 +1807,10 @@ export default {
         message: `${cmd('session')} use is not supported in the ${cmd('switchToDomain')} callback. Consider using it outside of the callback instead. Otherwise, please 👍 the following issue and leave a comment with your use-case:`,
         docsUrl: 'https://on.cypress.io/github-issue/20721',
       },
+      Cypress_session: {
```

Comment:
Just curious, what are the reasons this can't be supported if we are expecting users to cy.session with cy.switchToDomain?

Reply:
It's not that it necessarily can't be supported. For the experimental release, we're choosing not the support it. We also don't know of any use-cases where it needs to be supported within the switchToDomain callback that won't work if it's used outside of the callback instead.